### DISCUSSION TITLE
Problem with batch testing results？

### DISCUSSION THREAD

**Discussion by 2077himasi** [projectdiscovery/httpx#1329] [Q&A]
httpx -l  1.txt  -content-length -status-code -title -cname -o 2.csv
Hi, When conducting batch testing, the results are mixed together, what are the methods. Make the title and cname in 2. CSV display separately in each column
For example, in 2. CSV, each row of a, b, and c is displayed
Easy to extract titles, etc

![image](https://github.com/projectdiscovery/httpx/assets/31888299/6c1e0000-dd28-4dd5-922d-13cc9d96ca53)
![image](https://github.com/projectdiscovery/httpx/assets/31888299/c701e925-35ad-454c-88d9-43150b91fa98)

**Comment by MetzinAround** [comment]
In order to output in csv you should use the `-csv` flag with the `-o` flag

**Reply by 2077himasi** [reply]
Thank you!!!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
